batch.py

```diff
@@ -217,15 +216,21 @@ def prepare_input_files(
     genotype matrix
     phenotype vector
     """
+
+    # check that variants exist for this gene, before importing
+    bim_file = to_path(genotype_file_bim)
+    if bim_file.stat().st_size == 0 or (not bim_file.exists()):
```

Comment:
does this just measure the size of the object? the file will always exist i think, just empty

Reply:
I've just reversed this condition - this `if` will be True if either the file does not exist or it contains 0 Bytes. I've just added the `exists` one in case empty plink results don't get written in future, in which case trying to measure the size of a missing file will fail